src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -2,6 +2,9 @@ import clsx from 'clsx';
 import React, { MouseEventHandler, ReactNode } from 'react';
 import styles from './ClickableStyle.module.css';
 
+type Variant = 'primary' | 'secondary' | 'icon' | 'link';
+type Status = 'brand' | 'neutral' | 'success' | 'warning' | 'error';
```

Comment:
nit: what do you feel about something like this:
```ts
export const VARIANTS = [
  'primary',
  'secondary',
  'icon',
  'link',
] as const;

export type Variant = typeof VARIANTS[number];
```
and similarly for `type Status`?

Not a change suggestion but wanted to get your thoughts on it since the exported `const VARIANTS` can be imported for use in the story files / other files but not sure how much use this will actually see and might just make the code look more complex

Reply:
I think that especially makes sense here because the `Button` and `Link` components use those values in their story `argTypes`. Updated in this PR 👍